templates/addons/placementGroups/linodePlacementGroup.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: LinodePlacementGroup
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  region: ${LINODE_REGION}
```

Comment:
can we add credentialsRef here

Reply:
Sure

Reply:
After making this updated when I apply the generated template, I get the following error

Error from server (BadRequest): error when creating "tst-full-8v.yaml": LinodePlacementGroup in version "v1alpha2" cannot be handled as a LinodePlacementGroup: strict decoding error: unknown field "spec.credentialsRef.region"